torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -828,17 +829,28 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 np_tensor_shp = np.array(tensor_shape)
                 np_mask_shp = np.array(mask_shape)
                 if np.sum(np_tensor_shp % np_mask_shp) == 0:
-                    multiplier = np_tensor_shp // np_mask_shp
-                    mask_bcst = g.call_function(target_wrappers.repeat, args=(mask, multiplier.tolist()))
+                    if isinstance(fill_value, (float,)):
+                        fill_real_value = fill_value
+                    elif fill_value.op == "get_attr":
+                        with unset_fake_temporarily():
+                            fill_value_attr = getattr(gm, fill_value.target)
+                            fill_real_value = fill_value_attr.item()
```

Comment:
Had an offline discussion. This is to handle the special case when `masked_fill.Tensor` is called with a 0-D fill value tensor. Maybe we can add some comments here and guard unsupported cases

Reply:
I think the unsupported cases should fire an issue for each so that can show on the roadmap

Reply:
I have revised to more general condition check 54f20ca107a3119627d471ce5d399d7105444f2a for all case of constant, support either scalar or `torch.tensor`. Also add comment to mention this.

For all our current tests, not encounter unsupported cases. Furthermore, it seems in `torch.masked_fill` API not support tensor fill value (only support scalar) directly, wondering how this **distilbert-base-uncased** model got this `masked_fill.Tensor` op